src/zarr_benchmarks/parse_json_for_plots.py

```diff
@@ -439,18 +439,27 @@ def create_read_write_plots(benchmarks_df: pd.DataFrame) -> None:
     )
 
 
-def create_all_plots(json_ids: list[str] | None = None) -> None:
-    """Create all plots. If json_ids aren't provided, process the latest benchmark results inside data/results.
+def create_all_plots(
+    json_ids: list[str] | None = None, example_results: bool | None = True
```

Comment:
```suggestion
    json_ids: list[str] | None = None, example_results: bool = True
```
Given the default value is `True`, does `None` need to be an option here?

Reply:
Sorry for the delay - I was teaching a course yesterday! You're right - we don't need the `None` option here, and in fact I think the default should be `False` (i.e. to process from `data/results`, rather than `/example_results`). I'll make a quick PR to change this line.